lib/default/default/bindings/client_keys.lua

```diff
@@ -0,0 +1,51 @@
+local awful = require("awful")
```

Comment:
I might be reading it wrong, but it seems like you created a 'default' directory inside lib, and then added another 'default' directory to it.

Can I ask what are the benefits of doing so over getting rid of one of these 'default' folders?

Reply:
that was my idiocy(caused by not checking how it works after copying "default" dir that i checked as a simple module in a regular installation, copying into the repo), already fixed, but haven't actually tested anything, will get back to it when more knowledgeable contributors tell how they see it merged.